src/Symfony/Component/VarDumper/Cloner/Data.php

```diff
@@ -14,7 +14,7 @@
 /**
  * @author Nicolas Grekas <p@tchwork.com>
  */
-class Data
+class Data implements \Serializable
```

Comment:
Is it need? I don't get why, can't the object be serialized without implementing this?